odoo-shippable/scripts/build-image.sh

```diff
@@ -208,6 +201,10 @@ do
 
     deactivate
 done
+
+# Add python3.7-dev for compatibility with travis.yml
+ln -s ${REPO_REQUIREMENTS}/virtualenv/python3.7 ${REPO_REQUIREMENTS}/virtualenv/python3.7-dev
```

Comment:
What happens if this is not present? Do you have any resulting log handy?

Reply:
Do you mean python3.7 folder not created?

We are creating it above of this script